packages/library/src/commonMain/kotlin/io/realm/Cancellable.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2020 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm
+
+interface Cancellable {
```

Comment:
We could do that. I would probably wait until we have the other use cases, as it would then be easier to evaluate if we should have a common root interface for all _resource_-like entities.